src/main/java/org/openstreetmap/atlas/checks/distributed/IntegrityCheckSparkJob.java

```diff
@@ -70,39 +70,35 @@
     private static final Switch<String> COUNTRIES = new Switch<>("countries",
             "Comma-separated list of country ISO3 codes to be processed", StringConverter.IDENTITY,
             Optionality.REQUIRED);
+    // Indicator key for ignored countries
+    private static final String IGNORED_KEY = "Ignored";
```

Comment:
Can you disable your IDE's parameter reordering and retry? Stylistic changes are making it harder to understand the actual difference between files.